core/templates/dev/head/pages/creator-dashboard-page/creator-dashboard-page.directive.html

```diff
@@ -309,7 +309,7 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
       <span ng-if="$ctrl.subscribersList.length > 1"
             style="margin-top: 14px; display: inline-block;"
             translate="I18N_TOTAL_SUBSCRIBERS_TEXT"
-            translate-values="{totalSubscribers: <[$ctrl.subscribersList.length]>}">>
+            translate-values="{totalSubscribers: $ctrl.subscribersList.length}">>
```

Comment:
To test:
- Create 3 accounts.
- Make 1st account have 2 subscribers
- Go to Subscribers tab of the creator's dashboard page.
![image](https://user-images.githubusercontent.com/16653571/61579430-98110200-ab22-11e9-96f9-5d841fa301a2.png)